src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
-export interface Figure {
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue'
+}
 
+export interface Figure {
+  shape: string;
```

Comment:
add type for `shape` as well

Reply:
create type for shape 
`type Shape = 'triangle' | 'circle' | 'rectangle';`
and use it in the code

Reply:
```suggestion
  shape: Shape;
```
not fixed